src/test/ccl/ut_cclut_mock.inc

```diff
@@ -436,10 +436,14 @@ end ;execute_mocks_happy_validations
 /**
     Confirms that mock tables get dropped for a group1 user.
 
-    Note that in conjuction with test_mock_tables_dropped_via_tdbexecute_for_group1_user this test demonstrates that the
+    Note that in conjunction with test_mock_tables_dropped_via_tdbexecute_for_group1_user this test demonstrates that the
     tdbexecute call gets performed successfully.
 */
 subroutine test_mock_tables_dropped_for_group1_user(null)
+    if (reqinfo->updt_id = 0.0)
```

Comment:
This was pulled in by a merge of older code. Is it actually necessary? 
An attempt to create a table in a not authenticated non-dba session will terminate the test case, no?